zap/src/main/java/org/parosproxy/paros/core/scanner/VariantMultipartFormParameters.java

```diff
@@ -73,20 +75,25 @@ public void setMessage(HttpMessage msg) {
         }
 
         String contentType = msg.getRequestHeader().getHeader(HttpHeader.CONTENT_TYPE);
-        if (contentType == null || !contentType.toLowerCase().startsWith("multipart/form-data")) {
+        if (contentType == null
+                || !contentType.toLowerCase(Locale.ROOT).startsWith("multipart/form-data")) {
```

Comment:
Added `Locale.ROOT` just as a "clean code" action.

Reply:
It's actually a bug, better use `HttpHeader.hasContentType(String)` or `getNormalisedContentTypeValue()`

Reply:
Worth adding a test.

Reply:
Happy to, but I don't really know ow the condition(s) when the missing Locale comes into play.

Reply:
Make the content-type all upper case and set the default locale to Turkish (e.g. `Locale.setDefault(new Locale.Builder().setLanguageTag("tr").build());`.